server/routerlicious/packages/routerlicious-base/src/nexus/services/deltaService.ts

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
Does nexus use this anywhere? This is part of http flow, so should not be part of nexus.